plutus-core/cost-model/create-cost-model/BuiltinMemoryModels.hs

```diff
@@ -72,8 +72,8 @@ builtinMemoryModels = BuiltinCostModelBase
   , paramMultiplyInteger                 = Id $ ModelTwoArgumentsAddedSizes $ OneVariableLinearFunction 0 1
   , paramDivideInteger                   = Id $ ModelTwoArgumentsSubtractedSizes $ ModelSubtractedSizes 0 1 1
   , paramQuotientInteger                 = Id $ ModelTwoArgumentsSubtractedSizes $ ModelSubtractedSizes 0 1 1
-  , paramRemainderInteger                = Id $ ModelTwoArgumentsSubtractedSizes $ ModelSubtractedSizes 0 1 1
-  , paramModInteger                      = Id $ ModelTwoArgumentsSubtractedSizes $ ModelSubtractedSizes 0 1 1
+  , paramRemainderInteger                = Id $ ModelTwoArgumentsLinearInY $ OneVariableLinearFunction 0 1
```

Comment:
Does this apply to PlutusV3 only, or all versions? I'd expect it to be V3 specific but there's nothing in this module specific to V3.